requirements.txt

```diff
@@ -1,3 +1,4 @@
+aiohttp
```

Comment:
Remove this from here, and instead add it as an optional dependency for the `googlegroups` extras listed in setup.py.

We'll need to allow import of this to fail as well when doing so, so similarly to how its done here:

```python
        try:
            from google.auth.transport import requests
        except:
            raise ImportError(
                "Could not import google.auth.transport's requests,"
                "you may need to run 'pip install oauthenticator[googlegroups]' or not declare google groups"
            )
```